src/main/kotlin/com/mdrobnak/lalrpop/psi/ext/LpSymbol.kt

```diff
@@ -16,13 +16,14 @@ abstract class LpSymbolMixin(node: ASTNode) : ASTWrapperPsiElement(node), LpSymb
         // Ignore unselected symbols
         if (childrenWithLeaves.first().elementType != LpElementTypes.LESSTHAN) return null
 
+        val mut = childrenWithLeaves.any { it.elementType == LpElementTypes.MUT }
```

Comment:
nit: It would be nice to name this `isMutable` to match the class property name, and to make it more readable.